lib/rules/comment-pattern/index.js

```diff
@@ -0,0 +1,48 @@
+// @ts-nocheck
+
+'use strict';
+
+const _ = require('lodash');
+const report = require('../../utils/report');
+const ruleMessages = require('../../utils/ruleMessages');
+const validateOptions = require('../../utils/validateOptions');
+
+const ruleName = 'comment-pattern';
+
+const messages = ruleMessages(ruleName, {
+	expected: (pattern) => `Expected comment to match specified pattern "${pattern}"`,
```

Comment:
```suggestion
	expected: (pattern) => `Expected comment to match pattern "${pattern}"`,
```

Let's drop "specified" as we show the pattern in the message. I think including the pattern is a good idea, but let's create a follow-up issue to do this consistently across the other `*-pattern` rules.

Reply:
Great. I can open an issue soon